api/graphql/schema.graphql

```diff
@@ -1823,14 +1825,19 @@ type Mutation {
   publishPool(id: ID!): Pool @guard @canFind(ability: "publish", find: "id")
   changePoolClosingDate(
     id: ID!
-    newClosingDate: DateTime!
-      @rename(attribute: "new_closing_date")
-      @rules(
-        apply: ["after:today"]
-        messages: [{ rule: "after", message: "UpdatePoolClosingDate" }]
-      )
-  ): Pool @guard @canFind(ability: "changePoolClosingDate", find: "id")
-  closePool(id: ID!): Pool @guard @canFind(ability: "closePool", find: "id")
+    newClosingDate: DateTime! @rename(attribute: "new_closing_date")
+  ): Pool
```

Comment:
Same as the comment above, as far as I can tell, we have a habit of using the rename directive to change to a real database field. This is not the case here which is confusing
Pre-existing I know

Reply:
Yeah, that makes sense. Updated as part of [update field name for closing date, fix tests](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9945/commits/9c997aa5100d93589a7aa87eb6cea257846fff61)